go.mod

```diff
@@ -301,41 +300,43 @@ replace (
 	github.com/golang/protobuf => github.com/golang/protobuf v1.5.4
 	github.com/grpc-ecosystem/grpc-gateway => github.com/grpc-ecosystem/grpc-gateway v1.16.0
 
+	gituh.com/argoproj/gitops-enine => github.com/sivchari/gitops-engine v0.0.0-20250207011704-28b43ab93d0c
```

Comment:
This line will be deleted when the gitops's PR is merged.

Reply:
Merged!

Reply:
@crenshaw-dev 
Replaced!